config/onprem-iso-fcc.yaml

```diff
@@ -114,7 +114,7 @@ storage:
                 SERVICE_BASE_URL=https://assisted-api.local.openshift.io:8443
                 DEPLOY_TARGET=onprem
                 SERVICE_CA_CERT_PATH=/data/nginx-selfsigned.crt
-                OPENSHIFT_INSTALL_RELEASE_IMAGE=quay.io/ocpmetal/ocp-release:v1.0.9.1
+                OPENSHIFT_INSTALL_RELEASE_IMAGE=quay.io/openshift-release-dev/ocp-release:4.6.0-rc.4-x86_64
```

Comment:
The Makefile has already moved to quay.io/openshift-release-dev/ocp-release:4.6.1-x86_64. 

Should we create a target in the Makefile to auto generate the ignition configs/template and onprem-environment, and have a check in one of the build targets to verify that it is up to date?

Reply:
I think that's a great idea :100: